src/java/fr/paris/lutece/portal/service/workflow/WorkflowService.java

```diff
@@ -99,9 +99,11 @@ public static WorkflowService getInstance( )
     {
         if ( _singleton == null )
         {
-            _singleton = new WorkflowService( );
+            synchronized ( WorkflowService.class )

```

Comment:
same remark here for synchronisation scope

Reply:
Corrected in another commit of the same PR.